website/docs/usage/visualizers.mdx

```diff
@@ -118,12 +118,16 @@ displacy.serve(doc, style="ent")
 
 import DisplacyEntHtml from 'images/displacy-ent2.html'
 
-<Iframe title="displaCy visualizer for entities" html={DisplacyEntHtml} height={180} />
+<Iframe
+  title="displaCy visualizer for entities"
+  html={DisplacyEntHtml}
+  height={180}
+/>
 
 The entity visualizer lets you customize the following `options`:
 
 | Argument | Description                                                                                                   |
-| -------- | ------------------------------------------------------------------------------------------------------------- |
+| -------- | ------------------------------------------------------------------------------------------------------------- | ------ |
 | `ents`   | Entity types to highlight (`None` for all types). Defaults to `None`. ~~Optional[List[str]]~~                 | `None` |
```

Comment:
```suggestion
| `ents`   | Entity types to highlight (`None` for all types). Defaults to `None`. ~~Optional[List[str]]~~                 |
```
(I realise that this PR isn't at fault here!)

Reply:
So revert the changes from [2613dbb](https://github.com/explosion/spaCy/pull/11974/commits/2613dbb45efbc5d1ccd4e402ff09c93a49404f02), is that correct?